docs/docs/persistent-volumes.md

```diff
@@ -1,34 +1,38 @@
 
-When you specify a local volume or volumes, tasks and their associated data are “pinned” to the node they are first launched on and will be relaunched on that node if they terminate. The resources the application requires are also reserved, and Marathon will implicitly reserve an appropriate amount of disk space (as declared in the volume via `persistent.size`) in addition to the sandbox `disk` size you specify as part of the AppDefinition. Local volumes enable stateful tasks because the task can be restarted without data loss.
+When you specify a local volume or volumes, tasks and their associated data are “pinned” to the node they are first launched on and will be relaunched on that node if they terminate. The resources the application requires are also reserved. Marathon will implicitly reserve an appropriate amount of disk space (as declared in the volume via `persistent.size`) in addition to the sandbox `disk` size you specify as part of your application definition.
 
-## Configuration options
+### Benefits of using local persistent volumes
 
-A persistent volume is configured using the following options:
+- All resources needed to run tasks of your stateful service are dynamically reserved, thus ensuring the ability to relaunch the task on the same node using the same volume when needed.
+- You don't need constraints to pin a task to a particular agent where its data resides
+- You can still use constraints to specify distribution logic
+- Marathon lets you locate and destroy an unused persistent volume if you don't need it anymore
 
-- `containerPath`: The path where your application will read and write data. This can currently be only relative and non-nested (`"data"` but not `"/data"` and not `"/var/data"` and not `"var/data"`). You will find an example of how to configure an application with persistent volumes and an absolute and nested path.
-- `mode`: The mode of the volume. Currently, `"RW"` is the only possible value.
-- `persistent.size`: The size of the persistent volume in MiBs.
+## Create an application with local persistent volumes
 
-Additionally, you need to set a `residency` in order to tell Marathon to setup a stateful application. Currently, the only valid option for this is
-```
-"residency": {
-  "taskLostBehavior": "WAIT_FOREVER"
-}
-```
+### Prerequisites
+
+In order to create stateful applications using local persistent volumes in Marathon, you need to set 2 command line flags that Marathon will use to reserve/unreserve resources and create/destroy volumes.
+
+- `--mesos_authentication_principal`: You can choose any that suits your needs. However, if you have set up ACLs on your Mesos master, this must be an authenticated and authorized prinicpal.
+- `--mesos_role`: This should be a unique role and will be used implicitly, that is, you don't need to configure the Mesos master via `--roles`.
+
+### Configuration options
+
+Configure a persistent volume with the following options:
```

Comment:
I removed the ###Example header because there's already a section for examples. Seemed confusing.